src/routes/safe/components/Balances/Collectibles/index.tsx

```diff
@@ -154,4 +162,40 @@ const Collectibles = (): React.ReactElement => {
   )
 }
 
-export default Collectibles
+const CollectiblesPage = (): React.ReactElement => {
+  const NFT_APPS_TAG = 'nft'
+  const { allApps, togglePin } = useAppList()
+  const nftApps = useMemo(() => allApps.filter((app) => app.tags?.includes(NFT_APPS_TAG)), [allApps])
+
+  const infoBar = (
+    <InfoAlert
+      id="collectiblesInfo"
+      title="Use Safe Apps to view your NFT portfolio"
+      text="Get the most optimal experience with Safe Apps. View your collections, buy or sell NFTs, and more."
+    />
+  )
+
+  return (
+    <Collectibles>
+      {infoBar}
```

Comment:
I think we should not display the infoBar if `nftApps.length === 0`

Reply:
Doch, we should! Some NFTs aren't in the payload at all, notably ERC1155 tokens. So the only way to see them is via an app. That's the whole point of this feature.

Reply:
But isnt't the `nftApps` list the list of apps to handle NFTs on that chain?
So if it's empty we would display a banner to use apps but we would offer no apps on that chain.